app/assets/javascripts/controllers/cloud_network/cloud_network_form_controller.js

```diff
@@ -57,4 +57,12 @@ ManageIQ.angular.app.controller('cloudNetworkFormController', ['$http', '$scope'
     $scope.angularForm.$setPristine(true);
     miqService.miqFlash("warn", "All changes have been reset");
   };
+
+  $scope.filterNetworkManagerChanged = function(id) {
+    miqService.sparkleOn();
+    $http.get('/cloud_network/cloud_tenants_by_ems/' + id).success(function(data) {
+      $scope.available_tenants = data.available_tenants;
+    });
```

Comment:
Thanks for the review!  I looked at cloud_subnet_form_controller.js and didn't see the pattern you're talking about, however - could you link me to an example?